src/librustc_llvm/ffi.rs

```diff
@@ -288,6 +288,9 @@ pub enum RelocMode {
     Static = 1,
     PIC = 2,
     DynamicNoPic = 3,
+    ROPI = 4,
+    RWPI = 5,
+    ROPI_RWPI = 6,
```

Comment:
If you won't have numbers in the C++ bindings, you should remove them here, btw.